pkg/service/replay/replay.go

```diff
@@ -711,3 +713,84 @@ func (r *Replayer) ProvideMocks(ctx context.Context) error {
 	<-ctx.Done()
 	return nil
 }
+
+// Normalise initiates the normalise process for normalising the test cases.
+func (r *Replayer) Normalise(_ context.Context, cfg *config.Config) error {
+	path := cfg.Path
+	testSets := viper.GetString("test-sets")
+	testCases := viper.GetString("test-cases")
+	testRun := viper.GetString("test-run")
+	r.logger.Info("Test cases and Mock Path", zap.String("path", path))
+	testRunPath := filepath.Join(path, "reports", testRun)
+	r.logger.Info("Test Run Folder", zap.String("folder", testRunPath))
+	// Get list of YAML files in the test run folder
+	files, err := os.ReadDir(testRunPath)
+	if err != nil {
+		utils.LogError(r.logger, err, "Failed to read directory")
+		return err
+	}
+	// Iterate over each YAML file in the test run folder
+	for _, file := range files {
+		if strings.HasSuffix(file.Name(), ".yaml") {
+			filePath := filepath.Join(testRunPath, file.Name())
+			// Read the YAML file
+			yamlData, err := os.ReadFile(filePath)
```

Comment:
why are you not using the test or report dbs..?

Reply:
Will look into this